docs/warnings.rst

```diff
@@ -27,7 +27,7 @@ script using the `warnings.filterwarnings` function as follows::
      >>> import warnings
      >>> from astropy.io import fits
      >>> warnings.filterwarnings('ignore', category=UserWarning, append=True)
-     >>> fits.writeto(filename, data, clobber=True)
```

Comment:
Maybe it is out of scope here but I argue that `overwrite` (or `clobber`) is not needed altogether. The `doctest` is skipped here, so having that keyword or not does not make any difference as far as the example is concerned. We should not encourage people to overwrite their data blindly by default.